csclient/csclient.go

```diff
@@ -37,6 +37,7 @@ var ServerURL = "https://api.jujucharms.com/charmstore"
 // Client represents the client side of a charm store.
 type Client struct {
```

Comment:
Yeah, it could definitely have been done that way, although it makes it a little more fragile. I went with keeping the existing API.